### ISSUE TITLE
Document using brigade-project chart for creating projects during deployment

### ISSUE THREAD

**Issue by frazerk** [brigadecore/brigade#896]
We're looking into using brigade to deploy a component of our application. All of the other components are deployed via helm, and deployments themselves will be managed using istio.

Ideally, we'd be able to deploy brigade itself, the rest of our app, and the project all via helm charts. Considering the `brigade-project` chart has been deprecated in favor of `brig project create`, what is the recommended method of creating a project during a `helm install` deployment?

Output of `brig version`: v1.0.0-23-g1b849be

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"archive", BuildDate:"2019-03-29T11:10:21Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:45:25Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): AKS, minikube

**Comment by kooba** [comment]
We've talked about this yesterday during standup and I promised to add my 2¢ here.

I've been using helm charts to deploy projects as well. There is only one reason I've done that and it's that I'm able to set values with `--set abc=xyz` at the deployment time and use that feature to usually set some secrets by referencing env vars like `--set github.token=$(GITHUB_TOKEN)`.

At the moment with `brig` I can either use file with `-f` flag to load project configuration from there or use interactive prompt to enter values manually. Ideally I would not want to set all values inside the file and have an option of providing some values on the fly which would satisfy my requirement. I guess adding that feature to brig would not be too complicated. 

While using helm chart for project creation is marked [The Old Way](https://docs.brigade.sh/topics/projects/) in the docs it seems to be working just fine and chart has been migrated to new org.
